### ISSUE TITLE
Towny is not charging plot upkeep

### ISSUE THREAD

**Issue by MilesBHuff** [SettleScape/server#116]
Towny is not charging town banks for plot upkeep.  This is causing far too much money to accumulate in the economy, and is making it too easy to buy new plots.

**Comment by MilesBHuff** [comment]
I have decided to backcalculate town taxes based on current town upkeeps, back to the time that upkeep was originally disabled.

**Comment by slankford** [comment]
Just throwing this out there, this may cause extreme bankruptcy for some towns. I'd hate to be the singular cause of all this stuff by being too extreme of an outlier.

**Comment by MilesBHuff** [comment]
It's not just you, Silas, and this will actually hit Delta harder than any town.  Negative balances are actually supported in Towny now;  people will be able to work their way out of debt safely.

**Comment by slankford** [comment]
Another side note, if this affects some of the newer/younger players, it could exacerbate that player base's already fragile retention rate

**Comment by MilesBHuff** [comment]
Yeah, the retention rate is a problem.  But then again, no server can reasonably expect to turn every visitor into a regular player.  About 1/6 of our players are regulars, and that's actually not bad imho.
Small towns have very low upkeeps.  Ugab and Delta are pretty extreme outliers.  Mithrandir only has 58 plots, for example, while Ugab has 366 and Delta has a whopping 679!  Backdating $58/day back 365 days is only $21k, which isn't that bad;  especially considering the payer already dropped $67,000 on that town to get it to 58 plots.
